pyproject.toml

```diff
@@ -18,7 +18,7 @@ name = "bliss-deblender"
 packages = [{include = "bliss"}]
 readme = "README.md"
 repository = "https://github.com/prob-ml/bliss"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Probably a new minor version would be better because these changes are so major, i.e. "0.2.0"